Document Node.js beta CPU time profiler (#21468)
DataDog/documentation: content/en/profiler/profile_types.md

Once profiling is enabled, the following profile types are collected for [supported Node.js versions][1]:

CPU (beta, v5.0.0+, v4.24.0+, v3.45.0+)
: The time each function spent running on the CPU, including JavaScript and native code.<br />

> Needs to be enabled by setting the environment variable `DD_PROFILING_EXPERIMENTAL_CPU_ENABLED=1`.

Reviewer: ```suggestion
Set the environment variable `DD_PROFILING_EXPERIMENTAL_CPU_ENABLED=1` to enable it.
```
